lib/src/tar/import.rs

```diff
@@ -351,28 +351,27 @@ impl<'a> Importer<'a> {
         };
         let objtype = objtype_from_string(objtype)
             .ok_or_else(|| anyhow!("Invalid object type {}", objtype))?;
-        match (objtype, is_xattrs, &self.state) {
-            (ostree::ObjectType::Commit, _, ImportState::Initial) => {
+        if is_xattrs && objtype != ostree::ObjectType::File {
+            return Err(anyhow!("Found xattrs for non-file object type {}", objtype));
```

Comment:
Here and below, I think you were looking for `bail!()`.

Reply:
I kind of prefer the explicitness of seeing `return` actually myself.